src/Request.js

```diff
@@ -50,6 +50,19 @@ export default class Request {
             .join('&');
     }
 
+    static format_payload (payload, content_type) {
+        if (typeof payload !== 'object') {
+            return payload;
+        }
+
+        switch (content_type) {
+            case 'application/json':
+                return JSON.stringify(payload);
+            case 'application/x-www-form-urlencoded': default:
```

Comment:
remove `default:`?

Reply:
remove  `case 'application/x-www-form-urlencoded'` if any other value of content_type other than `'application/json'` will do the following statements

Reply:
This part of the code was rewritten.